Source/Core/DolphinQt/GCMemcardManager.cpp

```diff
@@ -135,6 +139,22 @@ void GCMemcardManager::ConnectWidgets()
   }
 }
 
+void GCMemcardManager::LoadDefaultMemcards()
+{
+  for (int i = 0; i < SLOT_COUNT; i++)
```

Comment:
Due to the ternary, this is a meaningless const and may as well just be a literal 2.

Reply:
Should we remove the const from everywhere else then? I think that's a bit out of scope for this PR.